recipes/shfmt/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "shfmt" %}
+{% set version = "3.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mvdan/sh/archive/v{{ version }}.tar.gz
+  sha256: 4ce55f902c6c405f740357a1ccf095b605d14eb5b56d882c74bced4d6410d4ae
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  host:
+    - go 1.12.*
```

Comment:
@conda-forge/core IDK how go works. Does this line need to be changed?

Reply:
Yes, we have `{{ compiler('go') }}` as a Jinja2 function which should go into build